src/WebViewProxy.ts

```diff
@@ -0,0 +1,33 @@
+import { EmitterSubscription, NativeEventEmitter } from "react-native";
+import { IWebViewProxy } from './WebViewProxyTypes';
+
+const scriptMessageEmitter = new NativeEventEmitter(undefined);
```

Comment:
does this work on `iOS`  - we used to pass the NativeModule before 

https://github.com/discord/react-native-webview/blob/11.18.1-discord-1/src/addOnMessageListenerWithWebViewKey.ts#L5

Reply:
oh nvm just saw `.ios.ts`

Reply:
ah - it seems like we have .ios.ts/.android.ts + the default

Reply:
Yup! That's right. I did test message passing in both directions between the RN app and the iframe in the WebView on iOS and Android, so I can confirm that this still works on iOS

Reply:
👍